ut.bat

```diff
@@ -1,5 +1,5 @@
 @echo off
 :: Pipes a diff against develop to the clipboard.
 :: The diff can then be pasted into the UnitTestDoc for testing.
-git diff develop | clip
+git diff origin/develop | clip
```

Comment:
Fixed a bug with the ut batch script. It should have been comparing against origin/develop and not the local develop.